src/sst/elements/vanadis/lsq/vbasiclsq.h

```diff
@@ -84,7 +84,7 @@ class VanadisBasicLoadStoreQueue : public SST::Vanadis::VanadisLoadStoreQueue {
             new StandardMem::Handler<SST::Vanadis::VanadisBasicLoadStoreQueue>(
                 this, &VanadisBasicLoadStoreQueue::processIncomingDataCacheEvent));
 
-        address_mask = params.find<uint64_t>("address_mask", 0xFFFFFFFFFFFFFFFFULL);
+        address_mask = params.find<uint64_t>("address_mask", 0xFFFFFFFFULL);
```

Comment:
The default should be full 64bits shouldn't it? This would be the equivalent to an empty mask. Once we have VM working, we don't want any mask right?